include/deal.II/meshworker/scratch_data.h

```diff
@@ -1330,6 +1559,297 @@ namespace MeshWorker
     return ret;
   }
 
+
+
+  template <int dim, int spacedim>
+  template <typename Extractor, typename Number>
+  const std::vector<typename FEInterfaceViews::View<dim, spacedim, Extractor>::
+                      template solution_value_type<Number>> &
+  ScratchData<dim, spacedim>::get_jumps_in_values(
+    const std::string &global_vector_name,
+    const Extractor &  variable,
+    const Number       dummy)
+  {
+    const std::vector<Number> &independent_local_dofs =
+      get_local_dof_values(global_vector_name, dummy);
+
+    const FEInterfaceValues<dim, spacedim> &feiv =
+      get_current_interface_fe_values();
+
+    AssertDimension(independent_local_dofs.size(),
+                    feiv.n_current_interface_dofs());
+
+    const unsigned int n_q_points = feiv.n_quadrature_points;
+
+    const std::string name = get_unique_name(
+      global_vector_name, variable, "_jump_values_q", n_q_points, dummy);
+
+    // Now build the return type
+    using RetType =
+      std::vector<typename FEValuesViews::View<dim, spacedim, Extractor>::
+                    template solution_value_type<Number>>;
+
+    RetType &ret =
+      internal_data_storage.template get_or_add_object_with_name<RetType>(
+        name, n_q_points);
+
+    AssertDimension(ret.size(), n_q_points);
+
+    feiv[variable].get_jump_in_function_values_from_local_dof_values(
+      independent_local_dofs, ret);
+    return ret;
+  }
```

Comment:
This is a copy-paste-adjust of https://github.com/dealii/dealii/blob/master/include/deal.II/meshworker/scratch_data.h#L1036-L1068. As is documented in the `reinit()` function for interfaces, `get_local_dof_values()` already returns all interface dofs. So,
- as part of the return type, `typename FEValuesViews::View<dim, spacedim, Extractor>` -> `typename FEInterfaceViews::View<dim, spacedim, Extractor>`
- the `FEValues` object just needed to be changed to `FEInterfaceValues`
- the string passed to get_unique_name `"_values_q"` -> `"_jump_values_q"`
- the final call `fev[variable].get_function_values_from_local_dof_values()` -> `feiv[variable].get_jump_in_function_values_from_local_dof_values()`

The same reasoning and patterns were applied to the other method implementations.